Errors and warning whenever running download service

**Describe the bug**

Running download service there are some info stating there is `[WARN]` or  `[ERROR]` info.


**Which UI**
none :)

**Hardware / Software**
Doesn't matter

**Steps to Reproduce**
1. `docker compose --profile download up --build`
2. Boom...
```
Attaching to webui-docker-download-1
webui-docker-download-1  | Downloading, this might take a while...
webui-docker-download-1  | 
webui-docker-download-1  | **10/22 20:46:12 [WARN] Unknown option:   # this is the only way aria2c won't fail if the file already exists**
webui-docker-download-1  | 
webui-docker-download-1  | 10/22 20:46:12 [NOTICE] Downloading 7 item(s)
webui-docker-download-1  | 
webui-docker-download-1  | **10/22 20:46:12 [ERROR] Exception caught while loading DHT routing table from /root/.cache/aria2/dht.dat**
webui-docker-download-1  | **Exception: [DHTRoutingTableDeserializer.cc:82] errorCode=1 Failed to load DHT routing table from /root/.cache/aria2/dht.dat**
webui-docker-download-1  | 
webui-docker-download-1  | 10/22 20:46:12 [NOTICE] IPv4 DHT: listening on UDP port 6958
webui-docker-download-1  | 
webui-docker-download-1  | 10/22 20:46:12 [NOTICE] IPv4 BitTorrent: listening on TCP port 6943
webui-docker-download-1  | 
webui-docker-download-1  | **10/22 20:46:12 [ERROR] IPv6 BitTorrent: failed to bind TCP port 6943**
webui-docker-download-1  | Exception: [SocketCore.cc:312] errorCode=1 Failed to bind a socket, cause: Name does not resolve
```
Despite all those scary warnings and errors it still manages to pull it off and download everything, so I would treat is as low priority.
First `[WARN]` could be corrected by not inserting comment there (aria2c manual states that line is a comment if it starts with `#`, but it seems that spaces before `#` sign are not ignored.
ipv6 could probably be omitted with `--disable-ipv6` option for aria2c. This error may be system specific as I've got ipv6 disabled on my box (if so, there should be method to verify ipv6 connectivity first to eventually provide proper options for aria2c).

--- a/services/download/download.sh
+++ b/services/download/download.sh
@@ -4,9 +4,14 @@ set -Eeuo pipefail
 
 mkdir -p /data/.cache /data/StableDiffusion /data/Codeformer /data/GFPGAN /data/ESRGAN /data/BSRGAN /data/RealESRGAN /data/SwinIR /data/LDSR /data/ScuNET /data/embeddings
 
+ARIA2C_OPTS=""
+if [ "`ip -6 addr`" == "" ]; then
+    ARIA2C_OPTS="${ARIA2C_OPTS} --disable-ipv6"
+fi
+
 echo "Downloading, this might take a while..."
 
-aria2c --input-file /docker/links.txt --dir /data --continue
+aria2c ${ARIA2C_OPTS} --input-file /docker/links.txt --dir /data --continue
 
 echo "Checking SHAs..."
 

--- a/services/download/download.sh
+++ b/services/download/download.sh
@@ -4,14 +4,9 @@ set -Eeuo pipefail
 
 mkdir -p /data/.cache /data/StableDiffusion /data/Codeformer /data/GFPGAN /data/ESRGAN /data/BSRGAN /data/RealESRGAN /data/SwinIR /data/LDSR /data/ScuNET /data/embeddings
 
-ARIA2C_OPTS=""
-if [ "`ip -6 addr`" == "" ]; then
-    ARIA2C_OPTS="${ARIA2C_OPTS} --disable-ipv6"
-fi
-
 echo "Downloading, this might take a while..."
 
-aria2c ${ARIA2C_OPTS} --input-file /docker/links.txt --dir /data --continue
+aria2c --disable-ipv6 --input-file /docker/links.txt --dir /data --continue
 
 echo "Checking SHAs..."
 

--- a/services/download/links.txt
+++ b/services/download/links.txt
@@ -1,10 +1,10 @@
+# 'check-integrity=true' is the only way aria2c won't fail if the file already exists
 magnet:?xt=urn:btih:2daef5b5f63a16a9af9169a529b1a773fc452637&dn=v1-5-pruned-emaonly.ckpt&tr=udp%3a%2f%2ftracker.opentrackr.org%3a1337%2fannounce&tr=udp%3a%2f%2f9.rarbg.com%3a2810%2fannounce&tr=udp%3a%2f%2ftracker.openbittorrent.com%3a6969%2fannounce&tr=udp%3a%2f%2fopentracker.i2p.rocks%3a6969%2fannounce&tr=https%3a%2f%2fopentracker.i2p.rocks%3a443%2fannounce&tr=http%3a%2f%2ftracker.openbittorrent.com%3a80%2fannounce&tr=udp%3a%2f%2ftracker.torrent.eu.org%3a451%2fannounce&tr=udp%3a%2f%2fopen.stealth.si%3a80%2fannounce&tr=udp%3a%2f%2fvibe.sleepyinternetfun.xyz%3a1738%2fannounce&tr=udp%3a%2f%2ftracker2.dler.org%3a80%2fannounce&tr=udp%3a%2f%2ftracker1.bt.moack.co.kr%3a80%2fannounce&tr=udp%3a%2f%2ftracker.zemoj.com%3a6969%2fannounce&tr=udp%3a%2f%2ftracker.tiny-vps.com%3a6969%2fannounce&tr=udp%3a%2f%2ftracker.theoks.net%3a6969%2fannounce&tr=udp%3a%2f%2ftracker.publictracker.xyz%3a6969%2fannounce&tr=udp%3a%2f%2ftracker.monitorit4.me%3a6969%2fannounce&tr=udp%3a%2f%2ftracker.moeking.me%3a6969%2fannounce&tr=udp%3a%2f%2ftracker.lelux.fi%3a6969%2fannounce&tr=udp%3a%2f%2ftracker.dler.org%3a6969%2fannounce&tr=udp%3a%2f%2ftracker.army%3a6969%2fannounce
   select-file=1
   index-out=1=StableDiffusion/v1-5-pruned-emaonly.ckpt
   follow-torrent=mem
   follow-metalink=mem
   seed-time=0
-  # this is the only way aria2c won't fail if the file already exists
   check-integrity=true
 https://drive.yerf.org/wl/?id=EBfTrmcCCUAGaQBXVIj5lJmEhjoP1tgl&mode=grid&download=1
   out=StableDiffusion/model.ckpt

